docs/Development.md

```diff
@@ -123,50 +123,23 @@ exporters/exporter/
 └── values.yaml
 ```
 
-### Running locally
-
-Running an exporter on your local machine should follow this process:
-
-1. Create a python virtual environment (Optional, but recommended)
-
-        python -m venv .venv
-        source .venv/bin/activate
-
-1. Install dependencies
-
-        pip install -r exporters/requirements.txt
-        pip install -r exporters/requirements-dev.txt
-
-1. Install the pelorus library
-
-        pip install exporters/
-
-1. Set any environment variables required (or desired) for the given exporter (see [Configuring Exporters](Configuration.md#configuring-exporters) to see supported variables).
-
-        export GIT_TOKEN=xxxx
-        export GIT_USER=xxxx
-
-1. Log in to your OpenShift cluster
-
-        oc login --token=<token> --server=https://api.cluster-my.fun.domain.com:6443 
+### Dev Environment Setup
 
-1. (Optional) To avoid certificate warnings and some possible errors, you need to set up your local machine to trust your cluster certificate
-
-    1.  Download your cluster ca.crt file
-    1.  Add cert to system trust bundle
-    1.  Pass cert bundle with your login command
+#### Python & Repo Setup
 
-            oc login --token=<token> --server=https://api.cluster-my.fun.domain.com:6443  --certificate-authority=/etc/pki/tls/certs/ca-bundle.crt
+After cloning the repo, you'll need python 3.9 installed.
```

Comment:
Re: 3.8, the code uses some 3.9+ features.

As for 3.10, it's just a matter of testing. We could see if there are good 3.10 S2I images readily available and target that if we want. I especially would love `match` :)

I'd also need to check if 3.10 is available in RHEL SCLs, though.

Targeting only one version is a good idea, because it prevents someone from accidentally introducing code that requires a newer version but isn't caught by any existing testing / CI (which we should improve anyway). I did exactly that previously 😬

Reply:
Let's get your makefile code merged and I'll see how I can use that to make this simpler.

Reply:
@KevinMGranger ok